index.js

```diff
@@ -12,7 +12,7 @@ module.exports = async path => {
     .trim()
     .split('\n')
     .pop()
-    .split(/\s+/)
+    .split(/\t+/)
```

Comment:
```suggestion
    .split(/\t+/)
```
```suggestion
    .split(/\t+/)
    .map(s => s.trim())
```

Adding this is necessary in order to also preserve the `unmount` functionality, otherwise:

```
  cmd: "hdiutil detach '/dev/disk2s2        '",
  stdout: '',
  stderr: 'hdiutil: detach failed - No such file or directory\n'
```